src/js/stores/app-store.js

```diff
@@ -685,6 +687,16 @@ var AppStore = Object.assign({}, EventEmitter.prototype, {
 
   getShowCreateArtifactDialog: () => _showCreateArtifactDialog,
 
+  getIntegrationVersion: function() {
+    // return version number
+    var version = '';
+    if (_IntegrationVersion) {
+      // if first character NaN, is master branch
+      version = isNaN(_IntegrationVersion.charAt(0)) ? 'master' : _IntegrationVersion;
```

Comment:
that is (at least almost) already accomplished with the `mender_environment.integrationVersion ? mender_environment.integrationVersion : 'master'` - so unless it's the integration version is set, it will default to "master" - so you can probably just reduce it to `getIntegrationVersion: () => _IntegrationVersion,`

Reply:
Great! Yes, I chose to be a conformist in this unfamiliar code territory. I didn't even look at what the code does! :-P